src/lib/y2packager/release_notes.rb

```diff
@@ -0,0 +1,61 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2017 SUSE LLC, All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+
+require "yast"
+
+module Y2Packager
+  # Release notes for a given product
+  #
+  # This class stores the content and some additional metadata about release
+  # notes for a given product.
+  class ReleaseNotes
+    # @return [String] Product name
```

Comment:
Just a note: is it the internal libzypp name (ID) or is it a human readable string like "SUSE Linux Enterprise Server 15"? It is not obvious at the first sight...